roles/django_app_k8s/tasks/deployments.yml

```diff
@@ -1,14 +1,16 @@
 ---
 
-- name: Set up the redis deployment
-  k8s:
-    state: present
+- name: Add bitnami helm repo redis
+  kubernetes.core.helm_repository:
+    name: bitnami
+    repo_url: "https://charts.bitnami.com/bitnami"
```

Comment:
shouldn't this be properly versioned? I see they're up to major version 17 already in the bitnami chart.